### ISSUE TITLE
After update, watchexec stopped working.

### ISSUE THREAD

**Issue by wdanilo** [watchexec/watchexec#484]
This is related: https://github.com/watchexec/cargo-watch/issues/241

It seems that newest `watchexec` changed the behavior and does not work anymore.

**Comment by wdanilo** [comment]
This happens even if I disable all ignore files:

<img width="774" alt="CleanShot 2023-01-20 at 20 24 07@2x" src="https://user-images.githubusercontent.com/1623053/213787775-9a90d45b-41b6-479a-afc9-96ccbaa55f2d.png">

Platform: macOS

**Comment by wdanilo** [comment]
After I add `--project-origin` option and changing a file, I've got more errors:

<img width="919" alt="CleanShot 2023-01-20 at 20 29 33@2x" src="https://user-images.githubusercontent.com/1623053/213788688-fd6a075d-b675-4339-a55e-21d649db0468.png">

**Comment by wdanilo** [comment]
It doesnt work with other shells neither: 

<img width="965" alt="CleanShot 2023-01-20 at 20 31 49@2x" src="https://user-images.githubusercontent.com/1623053/213789119-95e30f1f-6357-4b8f-b40d-87a992fee2d3.png">

**Comment by passcod** [comment]
I'm going to close this if you can't provide any actual detail.

**Comment by passcod** [comment]
Also, cargo-watch 8.3.0 uses a very old version of Watchexec, so any recent update of watchexec is either not related to cargo-watch, or it's something to do with a dependency.

But first, to get the `bug` label on this issue, you had to use [this template](https://github.com/watchexec/watchexec/blob/main/.github/ISSUE_TEMPLATE/bug_report.md), which asks for a bunch of things, and then you helpfully deleted all of it and provided nothing. I can't do anything with this.

**Comment by passcod** [comment]
Hey uh, why did you DELETE information from your post.

> Basically, after I bumped my installation of cargo-watch from 8.1.1 (which had issues - infinite looping, because it was not ignoring all things in .gitignore) to 8.3.0, no changes in source files (not ignored) are matched and cargo-watch basically does not work anymore for us.
>
> Also, I installed the newest watchexec, run it with watchexec --verbose "echo 'yo'" and changed a file, this is what I see:
>
> ![image](https://user-images.githubusercontent.com/155787/213839848-3c31b67f-ad5a-496a-9c29-b75e571c2515.png)

_that_ would tell me that this issue is a) not related to the other issue, and b) THAT SCREENSHOT HAS A TYPO in the command and that's your error there.

I'm going to close this ticket. I hope there's more information in the cargo-watch one that can help me help _you_.

**Comment by wdanilo** [comment]
@passcod, first of all, I'm sorry for lack of detail. You are right, I was in hurry and created this issue with a really bad description. Regarding the deleted screenshot, I removed it because I realized that I made typo while trying to debug why watchexec does not work. It was not the cause of the issue. I was not aware that `cargo-watch` does not use the newest version of this lib. This issues should be closed, I described what happens better here: https://github.com/watchexec/cargo-watch/issues/241
